jpo-geojsonconverter/src/main/resources/schemas/processed-tim.schema.json

```diff
@@ -0,0 +1,289 @@
+          "type" : "array",
+          "items" : {
+            "$ref" : "#/$defs/ProcessedTimContentItem"
+          }
+        },
+        "sentence" : {
+          "type" : "string"
+        },
+        "type" : {
+          "$ref" : "#/$defs/ProcessedContentType"
+        }
+      }
+    },
+    "ProcessedTimContentItem" : {
+      "type" : "object",
+      "properties" : {
+        "itisCode" : {
+          "type" : "integer",
+          "format" : "int64"
+        },
+        "itisPhrase" : {
+          "type" : "string"
+        },
+        "type" : {
+          "$ref" : "#/$defs/TimContentItemType"
+        }
+      }
+    },
+    "ProcessedTimFeature(Object)" : {
+      "type" : "object",
+      "properties" : {
+        "geometry" : { },
+        "id" : {
+          "type" : "integer",
+          "format" : "int32"
+        },
+        "properties" : {
+          "$ref" : "#/$defs/ProcessedTimProperties"
+        }
+      }
+    },
+    "ProcessedTimFeatureCollection" : {
+      "type" : "object",
+      "properties" : {
+        "features" : {
+          "type" : "array",
+          "items" : {
+            "$ref" : "#/$defs/ProcessedTimFeature(Object)"
+          }
+        }
+      }
+    },
+    "ProcessedTimProperties" : {
+      "type" : "object",
+      "properties" : {
+        "content" : {
+          "$ref" : "#/$defs/ProcessedTimContent"
+        },
+        "deploymentAgencyType" : {
+          "$ref" : "#/$defs/ProcessedDeploymentAgency"
+        },
+        "priority" : {
+          "type" : "integer",
+          "format" : "int32"
+        },
+        "regionInfoList" : {
+          "type" : "array",
+          "items" : {
+            "$ref" : "#/$defs/ProcessedRegionInfoBase"
+          }
+        },
+        "validityPeriod" : {
+          "$ref" : "#/$defs/ProcessedValidityPeriod"
+        }
+      }
+    },
+    "ProcessedValidationMessage" : {
+      "type" : "object",
+      "properties" : {
+        "exception" : {
+          "type" : "string"
+        },
+        "jsonPath" : {
+          "type" : "string"
+        },
+        "message" : {
+          "type" : "string"
+        },
+        "schemaPath" : {
+          "type" : "string"
+        }
+      }
+    },
+    "ProcessedValidityPeriod" : {
+      "type" : "object",
+      "properties" : {
+        "endTime" : {
+          "type" : "string",
+          "format" : "date-time"
+        },
+        "startTime" : {
+          "type" : "string",
+          "format" : "date-time"
+        }
+      }
+    },
+    "Standard" : {
+      "type" : "string",
+      "enum" : [ "ITWG", "CTW" ]
+    },
+    "TimContentItemType" : {
+      "type" : "string",
+      "enum" : [ "ITIS_CODE", "PLAIN_TEXT" ]
+    }
+  },
+  "type" : "object",
+  "properties" : {
+    "asn1" : {
+      "type" : "string"
+    },
+    "compliance" : {
+      "type" : "array",
+      "items" : {
+        "$ref" : "#/$defs/ProcessedCompliance"
+      }
+    },
+    "gnisRegionId" : {
```

Comment:
question: do we need the gnisRegionId? It's part of the packetID and can easily be pulled from the hex to int, for more compact storage it might be good to eliminate these types of easily calculable types from the schema.

Reply:
Okay, sure I will remove it.